locales/zh_CN/LC_MESSAGES/contributor_guide.po

```diff
@@ -595,146 +613,145 @@ msgstr "``[Bug]``: 修复 Bug;"
 
 #: ../../contributor_guide.rst:166
 msgid ""
-"Check out more tags in `misc/prtags.json <https://github.com/taichi-"
-"dev/taichi/blob/master/misc/prtags.json>`_."
+"Check out more tags in `misc/prtags.json <https://github.com/taichi-dev/taichi/"
+"blob/master/misc/prtags.json>`_."
 msgstr ""
+"在 `misc/prtags.json <https://github.com/taichi-dev/taichi/blob/master/misc/"
+"prtags.json>`_ 中查看更多标签."
 
 #: ../../contributor_guide.rst:167
-#, fuzzy
 msgid ""
-"When introducing a new tag, please update the list in "
-"``misc/prtags.json`` in the first PR with that tag, so that people can "
-"follow."
-msgstr "**在引进新标签时，请在第一个 PR 中用该标签更新列表，以便其余成员跟随使用。**"
+"When introducing a new tag, please update the list in ``misc/prtags.json`` in "
+"the first PR with that tag, so that people can follow."
+msgstr ""
+"在引进新标签时，请在首先使用该标签的 PR 中一并更新 ``misc/prtags.json`` 列表，以"
+"便其余成员跟随使用。"
 
 #: ../../contributor_guide.rst:171
 msgid ""
-"We do appreciate all kinds of contributions, yet we should not expose the"
-" title of every PR to end-users. Therefore the changelog will distinguish"
-" `what the user should know` from `what the developers are doing`. This "
-"is done by **capitalizing PR tags**:"
+"We do appreciate all kinds of contributions, yet we should not expose the title "
+"of every PR to end-users. Therefore the changelog will distinguish `what the "
+"user should know` from `what the developers are doing`. This is done by "
+"**capitalizing PR tags**:"
 msgstr ""
-"我们感谢所有的贡献，但是我们不应该把每一个 PR 的标题暴露给终端用户。因此，应该将变更日志分类成 `用户应该知道什么` 和 "
-"`开发人员正在做什么` 是必要的。而这是通过 **大写的 PR 标签** 区分的:"
+"我们感谢所有的贡献，但是我们不应该把每一个 PR 的标题暴露给终端用户。因此，应该将"
+"变更日志分类成 `用户应该知道什么` 和 `开发人员正在做什么` 是必要的。而这是通过 **"
+"大写的 PR 标签** 区分的:"
```

Comment:
```suggestion
"大写 PR 标签** 区分的:"
```

Reply:
This should be a verb, meaning make it uppercase ;-)

Reply:
That's very good advice!